### DISCUSSION TITLE
Game server usage

### DISCUSSION THREAD

**Discussion by kernpinoy** [coollabsio/coolify#2995] [Q&A] [answered]
I kinda have an OK VPS, 8gb x 8cpu, and I kinda want to host a Minecraft game. I deploy the gameserver via Coolify, right? I know how to use docker-compose, i dunno it that helps. 

Thanks for this amazing program!

**Comment by peaklabs-dev** [comment, marked as answer]
Absolutely possible, just try it and see if the server resources are enough. There is also a Minecraft service template available in the latest coolify v4.
